### PR TITLE
Emi.wordy fixes update

### PR DESCRIPTION
**Pull request by tenmaster**
This PR contains the changes suggested after the Wordy's web reviewed all the android quickstart documentation.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/android/00-introduction.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Introduction
 description: Short Introduction to the Auth0 Android Quickstarts.
 ---
 
-This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your android apps with Auth0.
+This is the very beginning of a simple, practical and multi-step Quickstarts that will guide you through managing authentication in your android apps with Auth0.
```
Comment (chenkie) [inline articles/native-platforms/android/00-introduction.md]: `Quickstarts` => `quickstart`

File: articles/native-platforms/android/01-login.md [hunk 2]
```diff
@@ -138,8 +140,8 @@ startActivity(this.lock.newIntent(this));
 
 ### Done!
 
-You've already implemented Login and Sign Up with Auth0 in your Android project!
+You've already implemented both login and sign up process with Auth0 in your Android project!
```
Comment (chenkie) [inline articles/native-platforms/android/01-login.md]: `sign up process` = `signup`

File: articles/native-platforms/android/00-introduction.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: Introduction
 description: Short Introduction to the Auth0 Android Quickstarts.
 ---
 
-This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your android apps with Auth0.
+This is the very beginning of a simple, practical and multi-step quickstarts that will guide you through managing authentication in your android apps with Auth0.
```
Comment (ntotten) [inline articles/native-platforms/android/00-introduction.md:6]: not correct, should be quickstart

File: articles/native-platforms/android/01-login.md [hunk 4]
```diff
@@ -74,9 +76,9 @@ Also, you need to add the following permissions:
 
 > Do not add ``<android:noHistory="true">`` to the ``LockActivity`` as this will alter the correct functionality of **Lock**.
 
-### 3. Implement the Login
+### 3. Implement The Login
```
Comment (ntotten) [inline articles/native-platforms/android/01-login.md]: this isnt really correct title casing should be `the`

File: articles/native-platforms/android/01-login.md [hunk 5]
```diff
@@ -138,8 +140,8 @@ startActivity(this.lock.newIntent(this));
 
 ### Done!
 
-You've already implemented Login and Sign Up with Auth0 in your Android project!
+You've already implemented both login and signup with Auth0 in your Android project!
 
-### Optional: Log In with Social Connections
+### Optional: Log In With Social Connections
```
Comment (ntotten) [inline articles/native-platforms/android/01-login.md]: same `with`

File: articles/native-platforms/android/03-session-handling.md [hunk 6]
```diff
@@ -49,8 +50,51 @@ private LockCallback callback = new AuthenticationCallback() {
...
+```java
+if(CredentialsManager.getCredentials(this).getIdToken() == null) {
+	// Prompt Login screen.
+} 
+else {
+	// Try to make an automatic login
+}
+```
+
+### 3. Validate an existent idToken
```
Comment (chenkie) [inline articles/native-platforms/android/03-session-handling.md]: `Validate an existant..` => `Validate an existing..`

File: articles/native-platforms/android/03-session-handling.md [hunk 7]
```diff
@@ -49,8 +50,51 @@ private LockCallback callback = new AuthenticationCallback() {
 };
 ```
 
+::: panel-info In the seed project, we use the `SharedPreference` object in private mode to store different `Credentials` values. There are other means of storage, but we won't cover them in this tutorial. Feel free to save them if you’d like.
+
+:::
+
+### 2. At Startup: Check idToken existence
```
Comment (chenkie) [inline articles/native-platforms/android/03-session-handling.md:57]: This should say: `Check for an Existing idToken at Startup`

**Comment by mpaktiti** [comment]
Hi @tenmaster I have updated the custom login md file in order to fix a bug: https://github.com/auth0/docs/pull/2190

Please make sure that you do not overwrite these changes with this PR. Thanks.

^ @chenkie

**Pull request merged**